airflow/providers/cncf/kubernetes/utils/pod_manager.py

```diff
@@ -70,15 +73,20 @@ class PodPhase:
     terminal_states = {FAILED, SUCCEEDED}
 
 
+def get_container_status(pod: V1Pod, container_name: str) -> V1ContainerStatus | None:
+    """Retrieves container status"""
+    container_statuses = pod.status.container_statuses if pod and pod.status else None
+    if container_statuses:
+        return next((x for x in container_statuses if x.name == container_name), None)
```

Comment:
Can you leave a short comment above this line explaining why you're using next. I get it, but it's a bit "deep python" and might come off as magic for someone just reading this the first time.

Reply:
Sure